### PR TITLE
Create How to support new model.md

### PR DESCRIPTION
**Pull request by matrixgame2018**
Translated the document into Chinese and create the doc in the  /docs/zh_cn/tutorials/How to support new model.md

### CODE REVIEW

File: docs/zh_cn/tutorials/How to support new model.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+## 如何支持新的模型
```
Comment (RunningLeon) [inline docs/zh_cn/tutorials/How to support new model.md:1]: Could you rename file to `how_to_support_new_models.md` with no empty space in filename?

File: docs/zh_cn/tutorials/How to support new model.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    if input.dim() == 1 and len(size) == 1:
+        return origin_func(input.unsqueeze(0), *([1] + list(size))).squeeze(0)
+    else:
+        return origin_func(input, *size)
+```
+
+使用函数重写器是十分容易的，只需添加一个带参数的装饰器即可：
+
+- `func_name`是需要被重载的函数，它可以是其他PyTorch 的函数或者是自定义的函数。模块中的方法也可以通过工具进行重载。
+-  `backend`是推理引擎。当模型被导入到引擎的时候，函数会被重载。如过没有给出，重载默认的参数就是重载的参数。如果后端的重载的参数不存在，将会按照预设的默认模式进行重载。
```
Comment (RunningLeon) [inline docs/zh_cn/tutorials/How to support new model.md:24]: typo: `如过`

File: docs/zh_cn/tutorials/How to support new model.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+就像函数重载器一样，可添加一个带参数的装饰器：
+
+- `module_type` 要重载的模块类。
+- `backend` 是推理引擎。当模型被导入到引擎的时候，函数会被重载。如过没有给出，重载默认的参数就是重载的参数。如果后端的重载的参数不存在，将会按照预设的默认模式进行重载。
+
+网络中模块的所有实例都将替换为这个新类的实例。原始模块和部署配置将作为前两个参数进行传递。
+
+### 自定符号标记
```
Comment (RunningLeon) [inline docs/zh_cn/tutorials/How to support new model.md:62]: 符号函数重写

File: docs/zh_cn/tutorials/How_to_support_new_model.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
+## 如何支持新的模型
```
Comment (RunningLeon) [inline docs/zh_cn/tutorials/How_to_support_new_model.md:1]: Better to change filename  from `How_to_support_new_model.md` to `how_to_support_new_models.md`, which is aligned with the English file.

File: docs/zh_cn/tutorials/How_to_support_new_model.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        return self._module(*args, **kwargs)
+    def init_weights(self, *args, **kwargs):
+        """Initialize weights."""
+        return self._module.init_weights(*args, **kwargs)
+```
+
+就像函数重载器一样，可添加一个带参数的装饰器：
+
+- `module_type` 要重载的模块类。
+- `backend` 是推理引擎。当模型被导入到引擎的时候，函数会被重载。如过没有给出，重载默认的参数就是重载的参数。如果后端的重载的参数不存在，将会按照预设的默认模式进行重载。
```
Comment (RunningLeon) [inline docs/zh_cn/tutorials/How_to_support_new_model.md:58]: typo: `如过`

File: docs/zh_cn/tutorials/How_to_support_new_model.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+                dims.append(i)
+    else:
+        dims = [sym_help._get_const(dim, 'i', 'dim')]
+    return g.op('Squeeze', self, axes_i=dims)
+```
+
+装饰器的参数
+
+- `func_name`要添加符号的函数名称。如果是自定义的，请使用完整路径`torch.autograd.Function`。或者如果它是 PyTorch 内置函数，则只用写一个名称即可。
+- `backend`是推理引擎。当模型被导入到引擎的时候，函数会被重载。如过没有给出，重载默认的参数就是重载的参数。如果后端的重载的参数不存在，将会按照预设的默认模式进行重载。
```
Comment (RunningLeon) [inline docs/zh_cn/tutorials/How_to_support_new_model.md:82]: typo: `如过`

File: docs/zh_cn/tutorials/How_to_support_new_model.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+- 如果函数是 PyTorch 内置函数，则为True。
+- `arg_descriptors` 符号函数参数的描述符，将被传递给`torch.onnx.symbolic_helper._parse_arg`。
+
+就像函数重载器的`ctx`一样，第一个参数会提供上下文信息。上下文中了一些有用的信息，例如部署配置ctx.cfg和原始功能（已被重载）`ctx.origin_func`。请注意， `ctx.origin_func`只能在 时使用`is_pytorch==False`。
```
Comment (RunningLeon) [inline docs/zh_cn/tutorials/How_to_support_new_model.md:86]: 只能在 `is_pytorch==False` 时使用

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by VVsssssk — APPROVED** [review]

**Pull request merged**